docs/REST API/rest-api.md

```diff
@@ -6376,32 +6376,70 @@ None.
 }
 ```
 
-### Get Apple Business Manager (ABM)
+### List Apple Business Manager (ABM) connections
 
 _Available in Fleet Premium_
 
-`GET /api/v1/fleet/abm`
+`GET /api/v1/fleet/apple_business_manager`
```

Comment:
Maintain support for the old `GET /abm` endpoint for backwards compatibility.

If there are multiple Apple Business Manager connections `GET /abm` returns the following error: TODO

Reply:
Redirect for the URL in the error message is here: https://github.com/fleetdm/fleet/pull/21043/files#diff-0c6120927d3e65309562b5b15b261d3298d05fcb0ff18e5b6512ee683b7ad6b3R532